### PR TITLE
chore: routingform org attribute seed

### PR DESCRIPTION
**Pull request by sean-brydon**
## What does this PR do?

This PR adds a way to seed a organization with routing forms and attirbutes. 

## Mandatory Tasks (DO NOT REMOVE)

- [ ] I have self-reviewed the code (A decent size PR without self-review might be rejected).
- [ ] I have updated the developer docs in /docs if this PR makes changes that would require a [documentation change](https://cal.com/docs). If N/A, write N/A here and check the checkbox.
- [ ] I confirm automated tests are in place that prove my fix is effective or that my feature works.

## How should this be tested?

Reset your DB. 
Run migrations
Run yarn seed (This should create an acme organization with now 10 users to make this a bit easier - you can change this value if you want more users in your org).
Run yarn seed-insights - This will seed the "Acme" org with a bunch of data. Previous bookings, Teams, Routing forms and attributes 

## Checklist

- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my changes generate no new warnings

### CODE REVIEW

File: yourProjectName.inlang/.gitignore
Comment (sean-brydon) [inline yourProjectName.inlang/.gitignore:1]: Will remove

File: packages/prisma/seed.ts [hunk 1]
```diff
@@ -1096,51 +1096,34 @@ async function main() {
...
-          },
-          inTeams: [
-            {
-              slug: "team1",
-              role: "ADMIN",
-            },
-          ],
-        },
-        {
+        ...Array.from({ length: 10 }, (_, i) => ({
```
Comment (sean-brydon) [inline packages/prisma/seed.ts:1099]: Creates 10 members . Feel free to change length to anything you want to test orgs

File: packages/prisma/seed-utils.ts [hunk 2]
```diff
@@ -264,3 +265,365 @@ export async function createTeamAndAddUsers(
...
+      id: true,
+      userId: true,
+    },
+  });
+
+  console.log(`🎯 Creating attributes for team ${teamId}`);
+
+  const attributeRaw: { id: string; options: { id: string; value: string }[] }[] = [];
+
+  for (const attr of mockAttributes) {
```
Comment (sean-brydon) [inline packages/prisma/seed-utils.ts:325]: Assign users random attributes

File: packages/prisma/seed-utils.ts [hunk 3]
```diff
@@ -264,3 +265,365 @@ export async function createTeamAndAddUsers(
...
+    name: `Seeded Form - Insights for ${teamId}`,
+    routes: [
+      {
+        id: "8a898988-89ab-4cde-b012-31823f708642",
+        value: "team/insights-team/team-javascript",
+      },
+      {
+        id: "8b2224b2-89ab-4cde-b012-31823f708642",
+        value: "team/insights-team/team-sales",
+      },
```
Comment (sean-brydon) [inline packages/prisma/seed-utils.ts:441]: Key info needed that can't change as its used in routingFormResponses

File: packages/prisma/seed-utils.ts [hunk 4]
```diff
@@ -264,3 +265,365 @@ export async function createTeamAndAddUsers(
...
+    ],
+    formFieldFilled: {
+      id: "83316968-45bf-4c9d-b5d4-5368a8d2d2a8",
```
Comment (sean-brydon) [inline packages/prisma/seed-utils.ts:444]: This formField that is filled when the users previews the attribute

File: packages/prisma/seed-utils.ts [hunk 5]
```diff
@@ -264,3 +265,365 @@ export async function createTeamAndAddUsers(
...
+          attributesQueryValue: {
+            id: "ab99bbb9-89ab-4cde-b012-319300f53ef8",
+            type: "group",
+            children1: {
+              "b98b98a8-0123-4456-b89a-b19300f55277": {
+                type: "rule",
+                properties: {
+                  field: attributeRaw[2].id,
+                  value: [
+                    attributeRaw[2].options.filter((opt) => opt.value === "JavaScript").map((opt) => opt.id),
```
Comment (sean-brydon) [inline packages/prisma/seed-utils.ts:486]: This currently relys on the set attributeArray being passed in. Down the line we can make this more flexible of a factory if we really need

File: packages/prisma/seed-utils.ts [hunk 6]
```diff
@@ -264,3 +265,365 @@ export async function createTeamAndAddUsers(
...
+        {
+          id: "148899aa-4567-489a-bcde-f1823f708646",
+          action: { type: "customPageMessage", value: "Fallback Message" },
+          isFallback: true,
+          queryValue: { id: "814899aa-4567-489a-bcde-f1823f708646", type: "group" },
+        },
+      ],
+      fields: [
+        {
+          id: seededForm.formFieldFilled.id,
```
Comment (sean-brydon) [inline packages/prisma/seed-utils.ts:541]: The field ID we use to fill in the form

File: packages/prisma/seed-utils.ts [hunk 7]
```diff
@@ -264,3 +265,365 @@ export async function createTeamAndAddUsers(
...
+          type: "multiselect",
+          label: "skills",
+          options: attributeRaw[2].options.map((opt) => ({
+            id: opt.id,
+            label: opt.value,
+          })),
```
Comment (sean-brydon) [inline packages/prisma/seed-utils.ts:547]: Make sure options and id are the same so we can link to responses

File: packages/prisma/seed-utils.ts [hunk 8]
```diff
@@ -264,3 +265,365 @@ export async function createTeamAndAddUsers(
...
+
+    const formFillerId = randomUUID();
+
+    return {
+      formId: seededForm.id,
+      formFillerId: formFillerId,
+      response: {
+        [seededForm.formFieldFilled.id]: {
+          label: "skills",
+          value: selectedSkills.map((opt) => opt.id),
```
Comment (sean-brydon) [inline packages/prisma/seed-utils.ts:617]: We can pass multiple responses in here Note: These responses may not match the rules that are created. They are just for visual and display purposes for insights.

File: packages/prisma/seed-insights.ts [hunk 9]
```diff
@@ -83,65 +85,37 @@ const shuffle = (
...
+      slug: "acme",
+      organizationSettings: {
+        isOrganizationVerified: true,
+        orgAutoAcceptEmail: "acme.com",
+        isAdminAPIEnabled: true,
+      },
     },
-  });
-
-  if (!insightsAdmin) {
```
Comment (sean-brydon) [inline packages/prisma/seed-insights.ts:93]: Note we may need to add this user back. I'm pretty sure it is used in a e2e test. Will check just wanted to get this out for early reviews

**Comment by hariombalhara** [comment]
Works great !!

**Comment by PeerRich** [comment]
test failing @sean-brydon image

File: apps/api/v1/test/lib/utils/retrieveScopedAccessibleUsers.integration-test.ts [hunk 10]
```diff
@@ -81,7 +81,7 @@ describe("retrieveScopedAccessibleUsers tests", () => {
         where: { email: "member2-acme@example.com" },
       });
 
-      expect(accessibleUserIds.length).toBe(3);
```
Comment (sean-brydon) [inline apps/api/v1/test/lib/utils/retrieveScopedAccessibleUsers.integration-test.ts:84]: Matches new count

**Review by keithwillcode — APPROVED** [review]

**Pull request merged**